content/en/tools/digital-currencies/digital-currencies-glossary.md

```diff
@@ -0,0 +1,44 @@
+ 
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: "Digital Currencies Glossary"
+    weight: 2000
+    parent: digital-currencies
+    identifier: digital-currencies-roles-glossary
+    description: "Digital Currencies documentation glossary."
+title: "Digital Currencies Glossary"
+ 
+
+** Central bank digital currency (CBDC)** A central bank-issued digital money, denominated in a national unit of account, which is backed by the full faith and credit of the issuing central bank.
+
+**Defining** Creating a token definition, which specifies the properties that a token of that type may haves. Defining a token is a similar process for CBDCs and for commerical bank-issued stablecoins; however, the approval process may differ.
+
+
+
+
+**Issuing party** The network party 
+
+**Redeeming** When a commercial bank or central bank removes a token, which it defined, from a transacting entity or a network.
+
+**Stablecoins** A digital money, issued by an officially chartered organization, such as a bank, that can be backed by fiat, a basket of assets, or faith in the issuer's balance sheet.
+
+**Token Operator**
+
+
+
+**Token Owner** Also Token Issuing Entity
+
+**Token Definition Authority** The role which must approve  any token definitions. There can be multiple users across the entity which reside in different dept. Their function is to review the token definition presented and approve or reject. Central Bank: Board of Governors. Commercial Bank: Head of Legal Services or Head of Regulation/Compliance.
+
+
+
+**Token Defining Entity** Token Defining Entity (TDE) – will author and intiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury
```

Comment:
```suggestion
**Token Defining Entity (TDE)** It will author and intiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury.
```

Reply:
Done.